src/content/docs/auto-telemetry-pixie/install-auto-telemetry-pixie.mdx

```diff
@@ -163,3 +167,12 @@ If you arrived in the guided installation process by following a link from Pixie
 ## If you link the wrong Pixie and New Relic account [#link-wrong-accounts]
 
 Contact support to unlink a Pixie account from your New Relic account. Be aware that if you unlink a Pixie account that was created automatically through the guided install, you'll lose access to that Pixie account.
+
+## Reduce ingest during install with Helm [#reduce-ingest-helm]
+
+If you want to reduce the amount of Pixie data that New Relic ingests, you can add the following parameters to your Helm chart during installation. Note that the data still exists in Pixie:
+
+  * `excludeNamespacesRegex` - use to set namespaces that you don't want to send observability data to New Relic. If empty, observability data for all namespaces is sent to New Relic. Example: `--set newrelic-pixie.excludeNamespacesRegex="examplenamespace-1|examplenamespace-2"`
```

Comment:
awkward phrasing there "use to set namespaces...." something is a bit off I think